tests/python/contrib/test_vitis_ai/test_vitis_ai_codegen.py

```diff
@@ -196,8 +213,13 @@ def expected():
 
 
 if __name__ == "__main__":
-    test_annotate()
-    test_add()
-    test_relu()
-    test_conv2d()
-    test_global_avg_pool2d()
+    if sys.platform == "win32":
+        print("Skip test on Windows for now")
+    else:
+        test_annotate()
+        test_add()
+        test_relu()
+        test_conv2d()
+        test_batchnorm()
+        test_global_avg_pool2d()
+        test_avg_pool2d()
```

Comment:
```suggestion
    if sys.platform == "win32":
        print("Skip test on Windows for now")
        return
    test_annotate()
    test_add()
    test_relu()
    test_conv2d()
    test_batchnorm()
    test_global_avg_pool2d()
    test_avg_pool2d()
```

Reply:
This actually gives a linter warning, that's why we switched to the if/else

Reply:
Interesting. What warning did you get exactly?

Reply:
Something like "Return statement in non-function body"

Reply:
Ah because `if __name__ == "__main__":` block is not in a function. You need to use `sys.exit(0)` instead of `return`.